kubernetes/ansible/roles/logging/templates/oauth2-proxy.yaml

```diff
@@ -41,3 +50,20 @@ authenticatedEmailsFile:
      {% endfor %}
 
 fullnameOverride: "oauth2-proxy"
+
+{% if (graylog_open_to_public is not defined or graylog_open_to_public is false) and (send_logs_to_graylog is defined and send_logs_to_graylog is true) and (graylog_open_to_private is defined and graylog_open_to_private is true) %}
+service:
+  type: LoadBalancer
+  loadBalancerIP: {{ oauth2_proxy_lb_ip }}
+  port: 80
+{% if nginx_private_ingress_annotations is defined and nginx_private_ingress_annotations %}
```

Comment:
we don't need this if condition here, as it's mandatory to give the annotation for private LB.

Reply:
If the user has already defined annotation for their private nginx, we can reuse the same variable here right? Else we wont be able to override in private as azure is hardcoded in the next section.

Reply:
:point_down: